src/dotnet-install.ps1

```diff
@@ -880,6 +895,20 @@ Say "- The SDK needs to be installed without user interaction and without admin
 Say "- The SDK installation doesn't need to persist across multiple CI runs."
 Say "To set up a development environment or to run apps, use installers rather than this script. Visit https://dotnet.microsoft.com/download to get the installer.`r`n"
 
+if ($Internal -and [string]::IsNullOrEmpty($FeedCredential)) {
+    if ($DryRun) {
+        Say-Warning "Provide credentials via -FeedCredential parameter."
```

Comment:
minor: it's possible these strings can get out of sync.  Should the string be declared once & then be used in both `Say-Warning` & `throw`?